data/hai missions.txt

```diff
@@ -1061,3 +1061,252 @@ mission "Hai Honeymoon"
+			`	You help her get the crates out of the cargo hold and contact the client. Nanachi's share of the payment is small, as the client transfers most of the money to the merchant she got the job from, but that doesn't make her any less excited about getting her first paycheck.`
+			`	"So, want to go to the spaceport and try to get a job on your own? I'll help you again in case you can't do it yet." you say.`
+			`	"Just give me an hour or two, back home our shipyard is kind of small, and I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here, I'll just go get a better look at them, I swear I'll be quick."`
+			`	You two agree to meet in the spaceport in a few hours, maybe this time she will do fine on her own.`
+
+
+
+mission "Nanachi 2"
+	name "Escort Nanachi to <planet>"
+	description "Bring Nanachi and her passengers to <destination>."
+	source "Cloudfire"
+	destination "Hai-home"
+	to offer
+		has "Nanachi 1: done"
+	on offer
+		conversation
+			`You meet up with Nanachi in the spaceport, give her some tips to try and persuade someone into hiring her, and tell her you'll be nearby if she still has trouble getting a job. You watch her go around the spaceport and approach some groups of humans and even other Hai to no avail, maybe since this is a military world, they aren't too keen on helping others as in the rest of their territory. She manages to grab the attention of a Hai family leaving a restaurant carrying way too much luggage, most likely tourists looking for a ride back home. Everything seems to be going well until the parents start to look worried. It seems as if they might go away if you don't step in.`
+			`	You walk over to the group to help her, and get a better look at the family: a Hai couple carrying a baby, and three older children who aren't doing much to help their father carry their ludicrous number of bags.`
+			choice
+				`	"Hey, what's going on?"`
+				`	"Can I be of assistance?"`
+			`	You introduce yourself and the parents tell you that they've been on vacation here and need transport back to Hai-home. "Don't take this the wrong way, but captain Nanachi told us her ship is one of your human ships, a Heavy Shuttle was it? It has enough bunks for us but our sons already caused a ruckus on the way here in a Hai ship, we're worried that since they aren't used to the accommodations, they might end up giving her too much trouble."`
+			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggage to Nanachi's ship. When the family is focused on a conversation, you quietly ask Nanachi what was the issue she was having with the other people she encountered.`
```

Comment:
Maybe shift from `going` to `traveling`?
`and the two of you` instead of `and you two`.

As for the issue, the player only knows that Nanachi was the common thread; therefore asking what issue she had seems reasonable.  If it was going to be changed, it would have to recognize that each of the people she talked to could have had different issues, so `what the issues were with the other people` or so.

Reply:
This was changed to "traveling" and "the two of you".  I feel the change from Going to Traveling is sufficient, as you're already convincing them with the start of the sentence.